driver/main.ml

```diff
@@ -188,6 +189,10 @@ let main () =
              are  incompatible with -pack, -a, -output-obj"
             (String.concat "|" P.pass_names)
     end;
+    if !noruntime && not !output_complete_object then begin
+      fatal
+        "-noruntime only applies to binaries produced by -output-complete-obj."
+    end;
```

Comment:
Is there any specific reason for this restriction ? I would have expected to be able to: 

```
ocamlopt -noruntime test.ml myruntimelib.a
ocamlopt -noruntime test.ml -cclib -lmyruntime
ocamlopt -noruntime -o test test.cmx myruntimelib.a
...
```

Reply:
Yes, I was not sure but this actually works. I removed the restriction.